atomata/capstone_WebClient
# Workbench page

### Summary 
-Container to display create button and load button.
-The load button container is scrollable. 
-Allows users to load, modify and save existing experiences. 
-Allows users to create and save new experiences.

### How to test?
1. Open Chrome. Press "F12" from keyboard to open Dev tool bar.
2. Search and click "Network" tab. Next checkbox the "Disable cache". 
3. Login with a new user name and password. 
4. Create a new experience. 
5. Add action on the action sequence list and click "save".
6. Click "return" and click the experience just created. 
7. Check if it loads accurately.
8. modify actions on the action sequence list and click "save" again.
9. Click "return" and click the experience again. 
10. Check if it loads accurately.
11. Click "return" and create another new experience. 
12. Repeat step 5 to step 10.

### Checks
